book/src/validator-stake.md

```diff
@@ -13,12 +13,29 @@ and use the wallet's `delegate-stake` command to stake your validator with 42 la
 $ solana-wallet delegate-stake ~/validator-config/stake-keypair.json ~/validator-vote-keypair.json 42
 ```
 
-Note that stake changes are applied at Epoch boundaries so it can take an hour
-or more for the change to take effect.
+Note that stakes need to warm up, and warmup increments are applied at Epoch boundaries, so it can take an hour
+or more for the change to fully take effect.
 
-Stake can be deactivate by running:
+Assuming your node is voting, now you're up and running and generating validator rewards.  You'll want
+to periodically redeem/claim your rewards:
+
+```bash
+$ solana-wallet redeem-vote-credits ~/validator-config/stake-keypair.json ~/validator-vote-keypair.json
```

Comment:
@rob-solana , it seemed to me during the dryrun that this fails with CustomError(1) (here [1]) if run multiple times, I'm guessing within the same epoch, although I haven't had a chance to test yet. Do you know if this is the case?
If so, can we please add a better error?

[1] https://github.com/solana-labs/solana/blob/32de5e6e7ae62c96610cd3c8831710b7fa6a4a57/programs/stake_api/src/stake_state.rs#L396

Reply:
all of staking errors are sux.  what's the best example to copy-pasta?

the failure you're seeing is likely due to there being no rewards available to collect

Reply:
so, if stake hasn't warmed up, or if you *just* collected rewards, there are no more rewards to collect until next epoch

Reply:
> all of staking errors are sux. what's the best example to copy-pasta?
A custom error struct like Budget or Token should work: https://github.com/solana-labs/solana/blob/master/programs/budget_api/src/budget_state.rs#L9-L25

Wallet already has a helper fn to decode.

Reply:
> so, if stake hasn't warmed up, or if you _just_ collected rewards, there are no more rewards to collect until next epoch

Thanks for clarifying!